Automattic/jetpack
# Subscribe block: add top margin to side settings w/ block theme

Fixes #29630

## Proposed changes:

This PR avoids layout issues when using a block theme and the Subscribe block.

| **Before** | **After** |
| ---------- | --------- |
| <img width="362" alt="Screenshot 2023-03-22 at 18 39 34" src="https://user-images.githubusercontent.com/426388/226992050-c4aba9a9-5e32-43da-ad77-4be7f62504ce.png"> | <img width="339" alt="Screenshot 2023-03-22 at 18 39 24" src="https://user-images.githubusercontent.com/426388/226992131-60579ed3-5ff7-401e-be5e-68877f6f7e0e.png"> |

### Other information:


## Jetpack product discussion

* No

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

We'll test this with 2 different themes:

1. Start with a site that's connected to WordPress.com, and where the Subscriptions feature is active. It's best if it's a site where you have a few subscribers.
2. Go to the Appearance > Themes menu and activate an old default theme such as Twenty Twenty.
3. Go to Posts > Add New
4. Insert a subscribe block.
5. In the sidebar, ensure the panel showing the number of subscribers is properly displayed.
6. Save this draft.
7. Go to Appearance > Themes, and switch to the Twenty Twenty Three theme
8. Go back to the post editor; the subscribe block settings should still look good (see screenshot above).
